data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -649,22 +635,35 @@ private void generateLineageDot() {
 		}
 		for (Node node : finalConstants) {
 			middle.append("\n" + node.toDotString());
-		}
+		}*/
+		middle.append("\n" + tableNode);
 		middle.append("\n");
-
+		//LOGGER.info("node id------"+tableNode.getNodeId());
+		if(targetNode != null) {
+			middle.append("\n" + targetNode);
+			middle.append("\n");
+		}
 		StringBuilder last = new StringBuilder();           // relations
-		for (Relation relation : finalRelations) {
-			if(relation.getSource() instanceof Column && relation.getDestination() instanceof Column){
+		/*for (Relation relation : finalRelations) {
+			if(relation.getSource() instanceof Column && relation.getDestination() instanceof Column) {
 			if (!(((Column) relation.getSource()).getTable().getTableName().equals(((Column) relation.getDestination()).getTable().getTableName()) && ((Column) relation.getSource()).getColumnName().equals(((Column) relation.getDestination()).getColumnName()))){
-				last.append("\n" + relation.toDotString());} }
+				last.append("\n" + relation.toDotString());}
+			}
 			else
 				last.append("\n" + relation.toDotString());
+		}*/
+		if (relationDotString != null) {
+			last.append("\n" + relationDotString);
+			last.append("\n}");
+		} else {
+			last.append("\n}");
 		}
-		last.append("\n}");
 
 		dotString = first.append(middle).append(last).toString();
+		System.out.println("printing dot string");
 		System.out.println("\n" + dotString + "\n");
-//		printLineageDot(dotString);
+		//printLineageDot(dotString);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)